web/src/api/__fixtures__/index/42.json

```diff
@@ -412,7 +413,12 @@
               "type": "boolean",
               "title": "Allow adding private repositories to the Hub",
               "default": false
-            }
+            },
+            "allowUserSignUp": {
+              "type": "boolean",
+              "title": "Allow new Users to Sign Up",
+              "default": true
+            },
```

Comment:
I think that the test is failing due to the trailing comma.

Reply:
Whoops, guess I've been spoiled by Dart :)

All checks are working fine now 👍